src/linkPanel/widgets/linking.ts

```diff
@@ -270,6 +296,26 @@ namespace Private {
     linkName: string;
   }
 
+  export function newLinkName(
+    basename: string,
+    sharedModel: IGlueSessionSharedModel
+  ): string {
+    let maxNum: number | undefined = undefined;
+    const re = RegExp(`^${basename}(?:_([0-9]+))?$`);
+    Object.keys(sharedModel.links).forEach(linkName => {
+      const match = re.exec(linkName);
+      if (match) {
+        if (!match[1] && !maxNum) {
+          maxNum = -1;
+        } else {
+          maxNum = Math.max(maxNum || -1, parseInt(match[1]));
+        }
+      }
+    });
+    const suffix = maxNum ? `_${maxNum + 1}` : '';
+    return `${basename}${suffix}`;
```

Comment:
What is this doing?

Could the link name be a simple UUID?

Reply:
Yes I guess, I was just following the way it works on Glue.
But it should work with any unique ID.

Reply:
Ok! You're right, let's do the same as Glue